app/src/test/java/io/intrepid/contest/screens/contestcreation/EditContestPresenterTest.java

```diff
@@ -0,0 +1,39 @@
+package io.intrepid.contest.screens.contestcreation;
+
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import io.intrepid.contest.models.Contest;
+import io.intrepid.contest.screens.contestcreation.EditContestContract;
+import io.intrepid.contest.testutils.TestPresenterConfiguration;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.verify;
+
+@RunWith(MockitoJUnitRunner.class)
+public class EditContestPresenterTest {
```

Comment:
We're also only at 21% coverage on this test, I'd like to see this get up much higher.